app/Http/Requests/PostRequest.php

```diff
@@ -42,6 +42,7 @@ public function rules()
             'text' => 'nullable|string|max:500',
             'location' => 'nullable|iso3166',
             'postal_code' => 'nullable|max:10',
+            'school_id' => 'nullable|string|max:255',
```

Comment:
I thought these were integers?

Reply:
Although we are using the numeric `universal-id` column as our School ID, we're saving as a [string](https://github.com/DoSomething/graphql/blob/99a98fd34e55c44b50ba97d3a644c9d253a1d84e/src/schema/schools.js#L15) for two reasons:
- support a `school-not-available` value to save if user selects that they can't find their school
- in case we ever want to use our own MongoDB ID's as identifiers (there's been talk about potentially querying colleges from a different dataset)

Reply:
Those are two very good reasons, good thinking!!